themes/solr/templates/htaccess.ref-guide-antora

```diff
@@ -0,0 +1,2 @@
+Redirect 302 /guide/solr/latest /guide/solr/9_0
+Redirect 302 /guide/solr/index.html /guide/solr/9_0/index.html
```

Comment:
Question: When I generated the `.htaccess` file by Antora (solr-ref-guide/build/site/solr/.htaccess), it generated a file like this:
```
Redirect 302 /guide/solr/latest /guide/solr/9_0
Redirect 301 /guide/index.html /guide/solr/9_0/index.html
```

The first one looks correct, but I cannot see how the second one is. The path `/guide/index.html` is currently occupied by our own website as a landing page for all guides (we'll still need that to link to pre-9.0 guides). Also, using a 301 permanent redirect for the index.html to a specific version looks fishy? That's why I changed it to 302 and the path in this PR.

Reply:
Thats wrong, yes! How does it come into play? Maybe the script needs some update?

Reply:
Antora has this built-in, I think it is this feature https://docs.antora.org/antora/latest/playbook/urls-redirect-facility/ but have not looked closely..

Reply:
I tried again, and Antora generates the same 301 rule again. No harm, RMs will just need to hand edit the file before committing to solr-site, until this is fixed. I opened https://issues.apache.org/jira/browse/SOLR-16188 for this, so we don't need to handle it here in this PR.